### DISCUSSION TITLE
More Aggresive "Screen Unlock" kinda.

### DISCUSSION THREAD

**Discussion by Ferdinand-33** [InfiniTimeOrg/InfiniTime#1893] [Ideas]
I curently have my watch set to only wake on shake with the sensitivity reasonably low. If i then touch the screen by accident(which happens reasonably frequently) i will end up in my apps or settings unintended. Its happend a few times where the brightness has changed or the watch has been set to silent or something similar without me actually changing it.

So my idea is the have the watch work as it currently does when you wake the watch it shows the default face and you are able to access the notification page. But then in order to access the apps and settings screens you have to double click the button(best method i can come up with).

**Comment by 42Tobias42** [comment]
Great Idea. I think just clicking the button just once would be enough. I dont think I click it by accident a lot

**Comment by FintasticMan** [comment]
See #1359.
